### PR TITLE
Add weekdays for month view and class comments

### LINKED ISSUE #53: Weekdays on top of month view columns
As a customer I want to see the weekdays on top of the month view columns so that I know which column belongs to which day. 

__

Als Kunde möchte ich, dass die Wochentage über den Spalten im Monatsview stehen, damit ich mich im Kalender besser zurecht finde.

estimated time: S

### PR DESCRIPTION
**Pull request by YoungBinKo**

### CODE REVIEW

File: packages/CalDAV-Core.package/CALMonthPanel.class/instance/buildDaysOfMonthWith..st [hunk 1]
```diff
@@ -4,6 +4,6 @@ buildDaysOfMonthWith: aBuilder
 	^ aBuilder pluggablePanelSpec new
 		model: self;
 		name: 'daysOfMonth';
-		frame: (0 @ self buttonHeight corner: 1 @ 1);
+		frame: (0 @ (self buttonHeight + 1/6) corner: 1 @ 1);
```
Comment (YoungBinKo) [inline packages/CalDAV-Core.package/CALMonthPanel.class/instance/buildDaysOfMonthWith..st]: should we put magic number (1/6) in a variable?

Comment (libaumann) [reply packages/CalDAV-Core.package/CALMonthPanel.class/instance/buildDaysOfMonthWith..st]: We could? That's basically the additional space so the weekdays fit between the buttons on top of the month view and the days of the month at the bottom. So maybe spaceForWeekdays?

Comment (Margarete01) [reply packages/CalDAV-Core.package/CALMonthPanel.class/instance/buildDaysOfMonthWith..st]: sounds good

**Review by libaumann — COMMENTED** [review]
I honestly don't know why but I had to change 1/6 to 1/12 for the correct layout when using the variable. Could someone try this out in their image to make sure it works?

File: packages/CalDAV-Core.package/CALCalendar.class/README.md [hunk 2]
```diff
@@ -1 +1,15 @@
...
+	color 					aColor
+
+color
+	- color attribute for the calendar
+
+name
+	- name of the calendar to describe it and distinguish it from other calendars 
+
+events 
+	- a SortedCollecting containing the events in the calendar
```
Comment (leanmas) [inline packages/CalDAV-Core.package/CALCalendar.class/README.md]: a SortedCollection*

File: packages/CalDAV-Core.package/CALCalendar.class/instance/color..st [hunk 3]
```diff
@@ -1,3 +1,3 @@
 accessing
-color: anObject
-	color := anObject
\ No newline at end of file
+color: aColor
```
Comment (leanmas) [inline packages/CalDAV-Core.package/CALCalendar.class/instance/color..st:2]: add empty line

File: packages/CalDAV-Core.package/CALCalendar.class/instance/events..st [hunk 4]
```diff
@@ -1,3 +1,3 @@
 accessing
-events: anObject
-	events := anObject
\ No newline at end of file
+events: aSortedCollection
```
Comment (leanmas) [inline packages/CalDAV-Core.package/CALCalendar.class/instance/events..st:2]: add empty line

File: packages/CalDAV-Core.package/CALCalendar.class/instance/name..st [hunk 5]
```diff
@@ -1,3 +1,3 @@
 accessing
-name: anObject
-	name := anObject
\ No newline at end of file
+name: aString
```
Comment (leanmas) [inline packages/CalDAV-Core.package/CALCalendar.class/instance/name..st:2]: add empty line

File: packages/CalDAV-Core.package/CALEditEventDialog.class/instance/oldEvent..st [hunk 6]
```diff
@@ -1,3 +1,3 @@
 accessing
-oldEvent: anObject
-	oldEvent := anObject
\ No newline at end of file
+oldEvent: anEvent
```
Comment (leanmas) [inline packages/CalDAV-Core.package/CALEditEventDialog.class/instance/oldEvent..st]: add empty line

File: packages/CalDAV-Core.package/CALEditEventDialog.class/instance/oldEvent..st [hunk 7]
```diff
@@ -1,3 +1,3 @@
 accessing
-oldEvent: anObject
-	oldEvent := anObject
\ No newline at end of file
+oldEvent: anEvent 
+	oldEvent := anEvent
```
Comment (leanmas) [inline packages/CalDAV-Core.package/CALEditEventDialog.class/instance/oldEvent..st]: also change anEvent to concrete Class (aCALEvent ?)

File: packages/CalDAV-Core.package/CALEvent.class/instance/beginDay..st [hunk 8]
```diff
@@ -1,3 +1,3 @@
 accessing
-beginDay: anObject
-	beginDay := anObject
\ No newline at end of file
+beginDay: aDay
```
Comment (leanmas) [inline packages/CalDAV-Core.package/CALEvent.class/instance/beginDay..st]: add empty line

File: packages/CalDAV-Core.package/CALEvent.class/instance/beginDay..st [hunk 9]
```diff
@@ -1,3 +1,3 @@
...
+	beginDay := aDay
```
Comment (leanmas) [inline packages/CalDAV-Core.package/CALEvent.class/instance/beginDay..st]: change to concrete Class (a CALDay ?)

File: packages/CalDAV-Core.package/CALEvent.class/instance/beginHour..st [hunk 10]
```diff
@@ -1,3 +1,3 @@
 accessing
-beginHour: anObject
-	beginHour := anObject
\ No newline at end of file
+beginHour: anHour
```
Comment (leanmas) [inline packages/CalDAV-Core.package/CALEvent.class/instance/beginHour..st]: add empty line

File: packages/CalDAV-Core.package/CALEvent.class/instance/beginMinute..st [hunk 11]
```diff
@@ -1,3 +1,3 @@
 accessing
-beginMinute: anObject
-	beginMinute := anObject
\ No newline at end of file
+beginMinute: aMinute
```
Comment (leanmas) [inline packages/CalDAV-Core.package/CALEvent.class/instance/beginMinute..st]: add empty line

File: packages/CalDAV-Core.package/CALEvent.class/instance/beginMonth..st [hunk 12]
```diff
@@ -1,3 +1,3 @@
 accessing
-beginMonth: anObject
-	beginMonth := anObject
\ No newline at end of file
+beginMonth: aMonth
```
Comment (leanmas) [inline packages/CalDAV-Core.package/CALEvent.class/instance/beginMonth..st]: add empty line

File: packages/CalDAV-Core.package/CALEvent.class/instance/beginYear..st [hunk 13]
```diff
@@ -1,3 +1,3 @@
 accessing
-beginYear: anObject
-	beginYear := anObject
\ No newline at end of file
+beginYear: aYear
```
Comment (leanmas) [inline packages/CalDAV-Core.package/CALEvent.class/instance/beginYear..st]: add empty line

File: packages/CalDAV-Core.package/CALEvent.class/instance/description..st [hunk 14]
```diff
@@ -1,3 +1,3 @@
 accessing
-description: anObject
-	description := anObject
\ No newline at end of file
+description: aString
```
Comment (leanmas) [inline packages/CalDAV-Core.package/CALEvent.class/instance/description..st:2]: add empty line

File: packages/CalDAV-Core.package/CALEvent.class/instance/endDay..st [hunk 15]
```diff
@@ -1,3 +1,3 @@
 accessing
-endDay: anObject
-	endDay := anObject
\ No newline at end of file
+endDay: aDay
```
Comment (leanmas) [inline packages/CalDAV-Core.package/CALEvent.class/instance/endDay..st]: add empty line

File: packages/CalDAV-Core.package/CALEvent.class/instance/endDay..st [hunk 16]
```diff
@@ -1,3 +1,3 @@
...
+	endDay := aDay
```
Comment (leanmas) [inline packages/CalDAV-Core.package/CALEvent.class/instance/endDay..st]: change to concrete Class (aCALDay ?)

File: packages/CalDAV-Core.package/CALEvent.class/instance/endHour..st [hunk 17]
```diff
@@ -1,3 +1,3 @@
 accessing
-endHour: anObject
-	endHour := anObject
\ No newline at end of file
+endHour: anHour
```
Comment (leanmas) [inline packages/CalDAV-Core.package/CALEvent.class/instance/endHour..st]: add empty line

File: packages/CalDAV-Core.package/CALEvent.class/instance/endMinute..st [hunk 18]
```diff
@@ -1,3 +1,3 @@
 accessing
-endMinute: anObject
-	endMinute := anObject
\ No newline at end of file
+endMinute: aMinute
```
Comment (leanmas) [inline packages/CalDAV-Core.package/CALEvent.class/instance/endMinute..st]: add empty line

File: packages/CalDAV-Core.package/CALEvent.class/instance/endMonth..st [hunk 19]
```diff
@@ -1,3 +1,3 @@
 accessing
-endMonth: anObject
-	endMonth := anObject
\ No newline at end of file
+endMonth: aMonth
```
Comment (leanmas) [inline packages/CalDAV-Core.package/CALEvent.class/instance/endMonth..st]: add empty line

File: packages/CalDAV-Core.package/CALEvent.class/instance/endYear..st [hunk 20]
```diff
@@ -1,3 +1,3 @@
 accessing
-endYear: anObject
-	endYear := anObject
\ No newline at end of file
+endYear: aYear
```
Comment (leanmas) [inline packages/CalDAV-Core.package/CALEvent.class/instance/endYear..st]: add empty line

File: packages/CalDAV-Core.package/CALEvent.class/instance/location..st [hunk 21]
```diff
@@ -1,3 +1,3 @@
 accessing
-location: anObject
-	location := anObject
\ No newline at end of file
+location: aString
```
Comment (leanmas) [inline packages/CalDAV-Core.package/CALEvent.class/instance/location..st:2]: add empty line

File: packages/CalDAV-Core.package/CALEvent.class/instance/title..st [hunk 22]
```diff
@@ -1,3 +1,3 @@
 accessing
-title: anObject
-	title := anObject
\ No newline at end of file
+title: aString
```
Comment (leanmas) [inline packages/CalDAV-Core.package/CALEvent.class/instance/title..st:2]: add empty line

File: packages/CalDAV-Core.package/CALEventDialog.class/instance/app..st [hunk 23]
```diff
@@ -1,3 +1,3 @@
 accessing
-app: anObject
-	parent := anObject
\ No newline at end of file
+app: aCALApp
```
Comment (leanmas) [inline packages/CalDAV-Core.package/CALEventDialog.class/instance/app..st:2]: add empty line

File: packages/CalDAV-Core.package/CALMonthPanel.class/instance/app..st [hunk 24]
```diff
@@ -1,3 +1,4 @@
 accessing
 app: anObject
```
Comment (leanmas) [inline packages/CalDAV-Core.package/CALMonthPanel.class/instance/app..st]: change anObject to concrete Class (aCALApp)

File: packages/CalDAV-Core.package/CALMonthPanel.class/instance/buildDayGrid..st [hunk 25]
```diff
@@ -4,8 +4,7 @@ buildDayGrid: aBuilder
 	fields := OrderedCollection new.
 	self fillMonth.
 	1 to: 6 do: [ :row |
-		 1 to: 7 do: [ :column |
-			fields add: (self buildDayWith: aBuilder row: row column: column).
+		 1 to: 7 do: [ :column | fields add: (self buildDayWith: aBuilder row: row column: column).
```
Comment (leanmas) [inline packages/CalDAV-Core.package/CALMonthPanel.class/instance/buildDayGrid..st]: remove space in between '[' and ':column'

Comment (leanmas) [inline packages/CalDAV-Core.package/CALMonthPanel.class/instance/buildDayGrid..st]: remove dot at end of last line

File: packages/CalDAV-Core.package/CALMonthPanel.class/instance/buildWeekDayOrderWith.day..st [hunk 26]
```diff
@@ -0,0 +1,12 @@
+toolbuilder
+buildWeekDayOrderWith: aBuilder day: aWeekday
+
+	| height width x y frame |
+	
+	width := 1 / 7.
```
Comment (leanmas) [inline packages/CalDAV-Core.package/CALMonthPanel.class/instance/buildWeekDayOrderWith.day..st]: outsource width and height into constant methods?

File: packages/CalDAV-Core.package/CALMonthPanel.class/instance/buildDayWith.row.column..st [hunk 27]
```diff
@@ -7,7 +7,7 @@ buildDayWith: aBuilder row: aRow column: aColumn
 	width := 1 / 7.
 	height := 1 / 6.
```
Comment (leanmas) [inline packages/CalDAV-Core.package/CALMonthPanel.class/instance/buildDayWith.row.column..st]: outsource width and height into constant methods?

File: packages/CalDAV-Core.package/CALMonthPanel.class/instance/weekdays.st [hunk 28]
```diff
@@ -0,0 +1,6 @@
+toolbuilder
+weekdays
+	| weekdays |
+	weekdays := OrderedCollection new. 
+	weekdays add: 'Monday'; add: 'Tuesday'; add: 'Wednesday'; add: 'Thursday'; add: 'Friday'; 				add: 'Saturday'; add: 'Sunday'. 
+	^weekdays
```
Comment (leanmas) [inline packages/CalDAV-Core.package/CALMonthPanel.class/instance/weekdays.st]: add space after ^

File: packages/CalDAV-Core.package/CALSubscriptionICSCalendar.class/instance/caldavLink..st [hunk 29]
```diff
@@ -1,3 +1,3 @@
 accessing
-caldavLink: anObject
-	caldavLink := anObject
\ No newline at end of file
+caldavLink: aLink
```
Comment (leanmas) [inline packages/CalDAV-Core.package/CALSubscriptionICSCalendar.class/instance/caldavLink..st]: what is aLink? might be aString? => maybe check Transcript to find out class of argument

File: packages/CalDAV-Core.package/CALEvent.class/instance/participants..st [hunk 30]
```diff
@@ -1,3 +1,3 @@
 accessing
-participants: anObject
-	participants := anObject
\ No newline at end of file
+participants: aString
```
Comment (leanmas) [inline packages/CalDAV-Core.package/CALEvent.class/instance/participants..st:2]: add empty line

**Review by leanmas — COMMENTED** [review]
few comments regarding code standard

File: packages/CalDAV-Core.package/CALMonthPanel.class/instance/buildDayGrid..st (hunk 25 above)
Comment (libaumann) [reply packages/CalDAV-Core.package/CALMonthPanel.class/instance/buildDayGrid..st]: we need the dot to be able to return fields or at least when I remove it fields turns red

**Pull request merged**